rasa/core/channels/mattermost.py

```diff
@@ -28,11 +28,11 @@ def token_from_login(cls, url: Text, user: Text, password: Text) -> Optional[Tex
         """Retrieve access token for mattermost user."""
 
         data = {"login_id": user, "password": password}
-        r = requests.post(url + "/users/login", data=json.dumps(data))
-        if r.status_code == 200:
-            return r.headers["Token"]
+        request = requests.post(url + "/users/login", data=json.dumps(data))
```

Comment:
This and the `r` below  in `__init__` actually refer to "response", not "request"

Reply:
Thanks @koernerfelicia , I got it wrong, I already made the changes with the correct names!